### PR TITLE
add cross env

### PR DESCRIPTION
**Pull request by salvoravida**
Hi!
i would like to add cross-env for windows env!

Hope you like it! :D

### CODE REVIEW

File: __tests__/all_spec.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ names.forEach((name) => {
     let delays;
 
     beforeAll(async () => {
-      await page.goto(`http://localhost:${port}/${name}`);
+      await page.goto(`http://localhost:${port}/${name}/index.html`);
```
Comment (dai-shi) [inline __tests__/all_spec.js]: Just curios, if you have any problem with the previous one?

File: package.json [hunk 2]
```diff
@@ -11,22 +11,22 @@
...
-    "build:unstated-next": "NAME=unstated-next webpack",
-    "build:zustand": "NAME=zustand webpack",
-    "build:react-sweet-state": "NAME=react-sweet-state webpack",
-    "build:storeon": "NAME=storeon webpack",
-    "build:react-hooks-global-state": "NAME=react-hooks-global-state webpack",
-    "build:use-context-selector": "NAME=use-context-selector webpack",
-    "build:mobx-react-lite": "NAME=mobx-react-lite webpack",
-    "build:use-subscription": "NAME=use-subscription webpack",
-    "build:salvoravida-react-redux": "NAME=salvoravida-react-redux webpack",
+    "jest": "cross-env PORT=8080 BABEL_ENV=jest jest",
```
Comment (dai-shi) [inline package.json]: Does this allow changing `PORT` from CLI?

File: package.json [hunk 3]
```diff
@@ -36,7 +36,7 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "@salvoravida/react-redux": "^7.1.6",
+    "@salvoravida/react-redux": "^7.1.8",
```
Comment (dai-shi) [inline package.json]: Huh, I will merge this change in advance.

File: .gitignore [hunk 4]
```diff
@@ -2,3 +2,4 @@
 *.swp
 node_modules
 /dist
+.idea/
```
Comment (dai-shi) [inline .gitignore]:
```suggestion
/.idea
```
Can this work? Is there a meaning in trailing slash?

File: __tests__/all_spec.js (hunk 1 above)
Comment (salvoravida) [reply __tests__/all_spec.js]: redirect loop

maybe a bug of http-server and windows?

File: .gitignore (hunk 4 above)
Comment (salvoravida) [reply .gitignore]: changed!

File: package.json (hunk 2 above)
Comment (salvoravida) [reply package.json]: i don't think so. anyway, i would like that you add cross-env, so i can merge future update without conflict

File: __tests__/all_spec.js (hunk 1 above)
Comment (dai-shi) [reply __tests__/all_spec.js]: gotcha.

File: package.json [hunk 5]
```diff
@@ -36,7 +36,7 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "@salvoravida/react-redux": "^7.1.8",
+    "@salvoravida/react-redux": "^7.1.9",
```
Comment (dai-shi) [inline package.json]:
```suggestion
    "@salvoravida/react-redux": "^7.1.8",
```
CI is failing. Let me do this later.

Comment (dai-shi) [reply package.json]: Oh, can you commit the suggestion?

Comment (salvoravida) [reply package.json]: done

**Comment by dai-shi** [comment]
Oh, cross-env... OK, merging this. I will adjust in master.

**Comment by dai-shi** [comment]
Thanks! Really appreciated.

**Pull request merged**